spec/index.html

```diff
@@ -733,125 +733,140 @@ <h3>Literals</h3>
+        Alternatively, an implementation MAY preserve the case from the original representation,
+        provided that it processes it in a case-insensitive manner.
+      </p>
+
+      <aside class=note>
+        The treatment of language tags has changed between RDF 1.1 and RDF 1.2.
+        In RDF 1.1, `"chat"@fr` and `"chat"@FR` theoretically represent two distinct terms, but implementations may replace one with the other via some form of normalization.
+        In RDF 1.2, they represent the exact same literal, i.e., the case difference in the concrete syntax does not propagate into the abstract syntax.
+        Since many RDF 1.1 implementations do normalize language tags internally, they will not be impacted by this change.
+      </aside>
+
+    </section>
+
+    <section>
+      <h2>Literal value</h2>
+
+        <p>The <dfn>literal value</dfn> associated with a <a>literal</a> is defined as follows.</p>
+
+      <ul>
+        <li>If the literal is a <a>language-tagged string</a>,
+          then the literal value is a pair consisting of its <a>lexical form</a>
+          and its <a>language tag</a>, in that order.</li>
+        <li>If the literal is a <a>directional language-tagged string</a>, then the literal value is
+          a tuple of its <a>lexical form</a>, its <a>language tag</a>, and its <a>base direction</a>,
+          likewise in that order.</li>
+        <li>If the literal's <a>datatype</a> is handled by an RDF implementation,
+          <ul>
+            <li>if the literal's <a>lexical form</a> is in the <a>lexical space</a>
+              of the <a>datatype</a>, then the literal value is the result of applying
+              the <a>lexical-to-value mapping</a> of the datatype to the
+              <a>lexical form</a>.</li>
+            <li>otherwise, the literal is <dfn data-lt-no-plural>ill-typed</dfn> and no literal value can be
+               associated with the literal. Such a case produces a semantic
+               inconsistency but is not <em>syntactically</em> ill-formed.
+               Implementations SHOULD accept [=ill-typed=] literals and produce RDF
+               graphs from them. Implementations MAY produce warnings when
+               encountering [=ill-typed=] literals.</li>
+          </ul>
+        </li>
+        <li>If the literal's <a>datatype IRI</a> is <em>not</em>
+          handled by an RDF implementation, then the literal value is
+          not defined by this specification. Implementations SHOULD accept
+          literals with unknown datatype IRIs and produce RDF graphs from them.
+        </li>
+      </ul>
+
+      <p>
+        Thus, two literals can have the same value
+        without being the same <a>RDF term</a>.
+        For example:</p>
+
+      <pre>
+        "1"^^xsd:integer
+        "01"^^xsd:integer
+      </pre>
+
+      <p>denote the same <a data-lt="literal value">value</a>, but are not the
+        same literal <a>RDF terms</a> because their
```

Comment:
```suggestion
        same literal <a>RDF term</a> because their
```

Reply:
The existing text says `same literal RDF terms` (i.e., plural), which I'm suggesting be changed to `same literal RDF term` (i.e., singular), which seems to me to conform to what [you're expressing](https://github.com/w3c/rdf-concepts/pull/162#discussion_r1991402727) you prefer?

Reply:
yes, sorry, I misread the diff.